Assets/Scripts/Timers/Timer.cs

```diff
@@ -38,6 +38,10 @@ public class Timer : MonoBehaviour
     /// Runs once when the timer stops.
```

Comment:
Small nitpick, but you can remove `System.Collections.Generic` and `System.Data` namespaces from this script.

Reply:
You should discuss that with the author @Stektpotet, as it's outside the scope of the changes I implemented in this PR